website/content/vagrant-cloud/api/v1.mdx

```diff
@@ -75,14 +75,14 @@ The resource you are trying to access does not exist. This may also be returned
 
 ##### **429** Too Many Requests
 
-You are currently being rate-limited. Please decrease your frequency of usage, or contact us at [support+vagrantcloud@hashicorp.com](mailto:support+vagrantcloud@hashicorp.com) with a description of your use case so that we can consider creating an exception.
+You are currently being rate-limited. Please decrease your frequency of usage, or  [contact support](/vagrant/vagrant-cloud/support) [contact support](/vagrant/vagrant-cloud/support) with a description of your use case so that we can consider creating an exception.
```

Comment:
```suggestion
You are currently being rate-limited. Please decrease your frequency of usage, or  [contact support](/vagrant/vagrant-cloud/support) with a description of your use case so that we can consider creating an exception.
```

Looks like there was a double link here

Reply:
Thanks for catching that! It got updated in my second commit when I swapped the main support section out of vagrantcloud and into the vagrant intro.